lib/app/extension/string_extension.dart

```diff
@@ -0,0 +1,16 @@
+extension StringExtension on String {
```

Comment:
Dart isn't supported auto import `Extension`.
Follow by: https://github.com/dart-lang/sdk/issues/38894

So I think should put `export 'string_extension.dart'` in `extension.dart` as a quick fix.

Just type `DigExtension` on the keyboard, then easier import `StringExtension`.

Reply:
Good idea